spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -257,6 +257,22 @@ Reloading the Public Key from the location at runtime as well as the frequency o
 reloading is beyond the scope of this specification and any such feature should be
 considered vendor-specific.
 
+#### `mp.jwt.token.header`
+
+The `mp.jwt.token.header` configuration property allows to set up the header which is expected to contain a JWT token.
+
+This configuration can either be `Authorization` (default), or `Cookie`. If a different configuration value is supplied,
+the configuration will be ignored and fallback to the default `Authorization`.
+
+Additional configuration options might be available in future versions of MP-JWT.
+
+#### `mp.jwt.token.cookie`
+
+The `mp.jwt.token.cookie` configuration property allows to set up the Cookie name which is expected to contain a JWT
+token.
+
+This configuration must be used together with `mp.jwt.token.header` set to `Cookie` or else it will be ignored.
```

Comment:
Please update as agreed at https://github.com/eclipse/microprofile-jwt-auth/pull/176#discussion_r397833042 :-)

Reply:
Or if you prefer to keep it as is - then I'll personally be fine :-)

Reply:
Ops, sorry. Missed that part. Done.